app/assets/javascripts/map/presenters/tabs/AnalysisNewPresenter.js

```diff
@@ -349,10 +349,50 @@ define([
 
       // SHAPE
       {
-        'Analysis/shape': function(useid, use, wdpaid) {
-          this.status.set('useid', useid);
-          this.status.set('use', use);
-          this.status.set('wdpaid', wdpaid);
+        'Analysis/shape': function(data) {
+          this.status.set({
+            useid: data.useid,
+            use: data.use,
+            wdpaid: data.wdpaid,
+          })
+        }
+      },
+      {
```

Comment:
We will take a look at this function in the future. Too much logic in an mps subscription.